deepchem/models/torch_models/tests/test_smiles2vec.py

```diff
@@ -77,9 +83,98 @@ def test_smiles2vec_model():
         n_tasks=n_tasks,
         max_seq_len=max_seq_len,
     )
-    model = Smiles2Vec(char_to_idx)
+    model = Smiles2Vec(char_to_idx=char_to_idx,
+                       max_seq_len=max_seq_len,
+                       n_tasks=n_tasks)
+
     input = torch.randint(low=0, high=len(char_to_idx), size=(1, max_seq_len))
     # Ex: input = torch.tensor([[32,32,32,32,32,32,25,29,15,17,29,29,32,32,32,32,32,32,32,32]])
 
     logits = model.forward(input)
     assert np.shape(logits) == (1, n_tasks, 1)
+
+
```

Comment:
Check if input/output for TF and Torch match with same weights

Reply:
yes I have added the test